modules/test/playwright/tests/data-set-manager/dataSets.spec.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {dataSetManagerPagesTest} from '../../fixtures/dataSetManagerPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(dataSetManagerPagesTest, loginTest);
+
+test('Data Set Test is created', async ({dataSetManagerPage, page}) => {
+	await dataSetManagerPage.goto();
+
+	await expect(
+		page.getByRole('link', {name: 'Data Set Test'}).first()
+	).toBeVisible();
+});
```

Comment:
I'm not sure about always creating test data. What if we want to have tests without data, or create different test data? It's probably better to have creation in a util, invoked `beforeAll` or `beforeEach`.

Also, `first` is probably not needed, I'm guessing it's a leftover from local testing.

Reply:
This point was clarified in meetings, so I'll close it here.

The best approach we found was using fetch functions placed in utils folder and invoke them in `beforeAll` clauses. 

In case we need to split between in-need-of from not-needing data tests, would use different files for them